front/src/modules/timesStops/consts.ts

```diff
@@ -0,0 +1,3 @@
+export const marginRegExValidation = /^\d+(\.\d+)?%$|^\d+(\.\d+)?min\/100km$/;
+
+export default marginRegExValidation;
```

Comment:
Disable the eslint warning here instead of default export the regex
```suggestion
export default marginRegExValidation;
```

Reply:
Done